spec/vcr/api/v1/short-form/submit/pre-lottery/with-supplemental-param.yml

```diff
@@ -45,17 +49,16 @@ http_interactions:
       - chunked
     body:
       encoding: ASCII-8BIT
-      string: '{"access_token":"<<ACCESS_TOKEN>>","instance_url":"https://<<SALESFORCE_HOST>>","id":"https://<<SALESFORCE_HOST>>/id/00D1D0000009nrYUAQ/005U00000066jl9IAA","token_type":"Bearer","issued_at":"1590532069542","signature":"DXXasvQj8CwCocjhGb7x7tfLBTy2CYuA7JJnOEOubiI="}'
-    http_version: null
-  recorded_at: Tue, 26 May 2020 22:27:49 GMT
+      string: '{"access_token":"<<ACCESS_TOKEN>>","instance_url":"https://<<SALESFORCE_HOST>>","id":"https://test.salesforce.com/id/00D1F000000I2yHUAS/005U00000066jl9IAA","token_type":"Bearer","issued_at":"1593681327685","signature":"5/hgfYw8iVreqIa2CIgS/9vJ4PmeohwrrvXs79cHWxg="}'
```

Comment:
Why not use the SALESFORCE_HOST variable here? curious because I think I was the one to make that change

Reply:
This is a VCR cassette it's auto generated.